models/cartDao.js

```diff
@@ -0,0 +1,146 @@
+const { appDataSource } = require("./appDataSource");
+
+const getItems = async (userId) => {
+  try {
+    const cartData = await appDataSource.query(
```

Comment:
```suggestion
    return appDataSource.query(
```
변수 선언 후 동기적으로 할당하는것이 아닌 곧바로 return 해주는것이 깔끔할 것 같습니다